packages/theme/src/theme/_visual-helpers.scss

```diff
@@ -176,3 +176,27 @@
 		}
 	}
 }
+
+@mixin btn-tertiary-colors($color: $btn-tertiary-default-color) {
+	&.btn-tertiary {
+		background-color: transparent;
+		border: none;
+		box-shadow: none;
+		font-weight: $btn-tertiary-font-weight;
+		line-height: $btn-tertiary-line-height;
+		min-height: $btn-tertiary-line-height;
+		padding: 0 $padding-small;
+		transition: background 0.2s ease-in;
+		color: $color;
+		&:hover {
+			color: $color;
+			background: tint($color, 90%);
+		}
+		&:active {
+			background: tint($color, 80%);
+		}
+		&:focus {
```

Comment:
Focus should have the same style as hover (accessibility)

Reply:
updated